README.md

```diff
@@ -110,6 +110,10 @@ In many rails projects when you run the server with `rails s` Rails sets RAILS_E
 
 Testing
 ========================
+First, set up a test database:
+bundle exec rake db:drop RAILS_ENV=test
+bundle exec rake db:create RAILS_ENV=test
```

Comment:
I think its important note that we're using a postgres database. The reason is, if your postgres test database isn't created, I think there might be possible issues when executing line 115. I'm not too certain but I think it would be a good idea to try the following process: Execute line 115 and make sure there isn't an already created db called test. What are the results?